test/org.hdfgroup.hdfview.test/uitest/TestHDFViewTAttr2.java

```diff
@@ -136,21 +136,21 @@ public class TestHDFViewTAttr2 extends AbstractWindowTest {
             { "0E", "10", "12" },
             { "14", "16", "18" } };
     String[][] referenceExpectedData = {
-            { "tattr2.h5/dset H5O_TYPE_OBJ_REF" }, { "tattr2.h5/dset H5O_TYPE_OBJ_REF" } };
```

Comment:
Was the file name removed from display? Seems like something you'd probably want to see.

Reply:
The difference is whether 1.10 based APIs or 1.12 new APIs are used. In this case it is 1.10 based.